.env.sample

```diff
@@ -19,3 +19,6 @@ WORKFLOWS_DEF_TOML_FILE=./resources/workflows.toml
 #MATRIX_SERVER_NAME=matrix.test:443
 #MATRIX_USERNAME=morpheus
 #MATRIX_PASSWORD=redpill
+
+# Bots
```

Comment:
Some explanation here would be nice, where we can point them to `resources/seeds/bots.go` to add a new bot